### PR TITLE
[FEATURE] Delete files fetched by a resource handler

### PR DESCRIPTION
**Pull request by IchHabRecht**
This feature allows to delete files that were fetched by a specific resource handler (e.g. delete all files fetched from placeholder.com).

### CODE REVIEW

File: Classes/Command/ResetCommand.php [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    {
+        parent::__construct($name);
+
+        $this->connection = $connection ?: GeneralUtility::makeInstance(ConnectionPool::class)->getConnectionForTable('sys_file');
+        $this->fileRepository = $fileRepository ?: GeneralUtility::makeInstance(FileRepository::class);
+    }
+
+    public function configure()
+    {
+        $this->setDescription('Deletes files fetched by filefill')
```
Comment (helhum) [inline Classes/Command/ResetCommand.php]: Wrong description

File: Classes/Command/ResetCommand.php [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            ->addOption(
+                'storage',
+                's',
+                InputOption::VALUE_OPTIONAL,
+                'Delete files from a specific storage only'
```
Comment (helhum) [inline Classes/Command/ResetCommand.php]: wrong description

File: Classes/Command/ResetCommand.php [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+                $expressionBuilder->eq(
+                    'f.storage',
+                    $queryBuilder->createNamedParameter($storage, \PDO::PARAM_INT)
+                )
+            );
+        }
+
+        $statement = $queryBuilder->execute();
+
+        while ($row = $statement->fetch(\PDO::FETCH_NUM)) {
```
Comment (helhum) [inline Classes/Command/ResetCommand.php]: if I understand this correctly, this resets missing files on all (or selected) storages, whether they are file fill enabled or not. is this correct?

Comment (IchHabRecht) [reply Classes/Command/ResetCommand.php]: Yes. It is necessary to be able to allow fetching files by filefill.

**Pull request merged**